components/form/fields/PasswordField/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { ChangeEventHandler, useState } from 'react'
+import { inputType } from '../../../../utilities/types/formTypes'
+import styles from './PasswordField.module.scss'
+import { FaEye, FaEyeSlash } from 'react-icons/fa'
+
+type PasswordFieldProps = {
+    field: inputType
+    id: string
+    name: string
+    placeholder: string
+    onChange: ChangeEventHandler
+    required?: boolean
+    minLength?: number
```

Comment:
Password field should require a minimum length. We will never have a password that doesnt have a min 😄

Reply:
i was thinking that and required to take off
and always leave it. makes sense